client/auth/token.go

```diff
@@ -0,0 +1,50 @@
+package auth
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+)
+
+var (
+	ErrInvalidToken = errors.New("invalid token")
+)
+
+type RawToken string
+
+// ParseExpires attempts to extract the `exp` claim from the JWT. This function
+// does not validate that the token is signed and should only be used if the
+// correctness of this value is not used for security.
+func (token RawToken) ParseExpires() (time.Time, error) {
```

Comment:
https://github.com/SKF/go-utility/blob/master/v2/jwt/jwt.go
 If you do not want to do the string parsing yourself.